app/assets/javascripts/discourse/lib/translation-plugin.js

```diff
@@ -20,7 +20,11 @@ class TranslationPlugin extends Plugin {
   }
 
   replaceMF(formats, input, path = []) {
-    Object.keys(input).forEach((key) => {
+    if (!input) {
+      return;
+    }
+
+    Object.keys(input || {}).forEach((key) => {
```

Comment:
`input || {}` seems unnecessary if you are short circuiting earlier?

Reply:
Ah, accidentally left that one in!